src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/cart/index.yaml

```diff
@@ -1,93 +1,89 @@
             body:
-                template: '@SyliusShop/cart/page/body/details/table/body.html.twig'
-            
-        'sylius_shop.cart.page.body.details.table.head':
+                template: '@SyliusShop/cart/page/form/sections/general/items/body.html.twig'
+
+        'sylius_shop.cart.page.form.sections.general.items.head':
             remove:
-                template: '@SyliusShop/cart/page/body/details/table/head/remove.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/head/remove.html.twig'
             item:
-                template: '@SyliusShop/cart/page/body/details/table/head/item.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/head/item.html.twig'
             unit_price:
-                template: '@SyliusShop/cart/page/body/details/table/head/unit_price.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/head/unit_price.html.twig'
             quantity:
-                template: '@SyliusShop/cart/page/body/details/table/head/quantity.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/head/quantity.html.twig'
             total:
-                template: '@SyliusShop/cart/page/body/details/table/head/total.html.twig'
-            
-        'sylius_shop.cart.page.body.details.table.body':
+                template: '@SyliusShop/cart/page/form/sections/general/items/head/total.html.twig'
+
+        'sylius_shop.cart.page.form.sections.general.items.body':
             remove:
-                template: '@SyliusShop/cart/page/body/details/table/body/remove.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/body/remove.html.twig'
             item:
-                template: '@SyliusShop/cart/page/body/details/table/body/item.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/body/item.html.twig'
             unit_price:
-                template: '@SyliusShop/cart/page/body/details/table/body/unit_price.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/body/unit_price.html.twig'
             quantity:
-                template: '@SyliusShop/cart/page/body/details/table/body/quantity.html.twig'
+                template: '@SyliusShop/cart/page/form/sections/general/items/body/quantity.html.twig'
             total:
-                template: '@SyliusShop/cart/page/body/details/table/body/total.html.twig'
-                
-        'sylius_shop.cart.page.body.sidebar':
+                template: '@SyliusShop/cart/page/form/sections/general/items/body/total.html.twig'
+
+        'sylius_shop.cart.page.form.sections.general#right':
             summary:
-                template: '@SyliusShop/cart/page/body/sidebar/summary.html.twig'
+                component: 'sylius_shop:checkout:summary'
```

Comment:
I wonder if it shouldn't be a cart/order summary instead of checkout 🤔

Reply:
That's a good question, I have decided to put this in the checkout directory because it will be used in each step of the checkout process.